woocommerce/woocommerce
# Update daily workflow to sharding

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Updates the daily testing workflows to execute end-to-end tests with sharding.

This PR also makes a fix to the reporting -- skipped tests have been reported higher than they should have because the Playwright blob reporter and allure reporters were conflicting.  This PR will correct that for all runs.





### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to Actions > Daily smoke test
2. Run the workflow using this branch ([here](https://github.com/woocommerce/woocommerce/actions/runs/7807131110) is a sample run if you don't want to wait).
3. Ensure workflow passes -- k6 tests are failing on daily currently, so they fail here too.  We'll look at fixing that in a different PR.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
